pkg/logql/shardmapper_test.go

```diff
@@ -328,12 +328,40 @@ func TestMappingStrings(t *testing.T) {
 			)`,
 		},
 		{
-			in:  `avg(avg_over_time({job=~"myapps.*"} |= "stats" | json busy="utilization" | unwrap busy [5m]))`,
-			out: `avg(avg_over_time({job=~"myapps.*"} |= "stats" | json busy="utilization" | unwrap busy [5m]))`,
+			in: `avg(avg_over_time({job=~"myapps.*"} |= "stats" | json busy="utilization" | unwrap busy [5m]))`,
+			out: `(
+				sum(
+					downstream<sum(avg_over_time({job=~"myapps.*"}|="stats"|jsonbusy="utilization"|unwrapbusy[5m])),shard=0_of_2>
+					++
```

Comment:
is this correct? `unwrapbusy` missing a space?

Reply:
and `jsonbusy`?

Reply:
We remove all white spaces in the tests. I've copied the expected string and adapted. Will fix.

Reply:
Done.